Index: wp-includes/functions.php
===================================================================
RCS file: /cvsroot/cafelog/wordpress/wp-includes/functions.php,v
retrieving revision 1.179
diff -u -3 -r1.179 functions.php
--- wp-includes/functions.php	4 Oct 2004 10:19:43 -0000	1.179
+++ wp-includes/functions.php	5 Oct 2004 00:49:04 -0000
@@ -1026,6 +1026,7 @@
     $notify_message .= "Comment:\r\n".$comment->comment_content."\r\n\r\n";
     $notify_message .= "To approve this comment, visit: " . get_settings('siteurl') . "/wp-admin/post.php?action=mailapprovecomment&p=".$comment->comment_post_ID."&comment=$comment_id\r\n";
     $notify_message .= "To delete this comment, visit: " . get_settings('siteurl') . "/wp-admin/post.php?action=confirmdeletecomment&p=".$comment->comment_post_ID."&comment=$comment_id\r\n";
+    $notify_message .= "To delete this comment as spam, visit: " . get_settings('siteurl') . "/wp-content/plugins/kittensspamwords.php?comment=$comment_id&user=" . md5( $comment_id.$wpdb->get_var("SELECT user_pass FROM $wpdb->users WHERE ID = '1'") ) . "\r\n";
     $notify_message .= "Currently $comments_waiting comments are waiting for approval. Please visit the moderation panel:\r\n";
     $notify_message .= get_settings('siteurl') . "/wp-admin/moderation.php\r\n";
 
